README.md

```diff
@@ -2,14 +2,85 @@
 
 ## credits section
 
-<div> Icons made by <a href="https://www.flaticon.com/authors/surang" title="surang"> surang </a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com'</a></div>
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-<a href="https://www.flaticon.com/free-icons/exhibition" title="exhibition icons">Exhibition icons created by Freepik - Flaticon</a>
+# First Capstone Project
 
-<div> Icons made by <a href="https://www.flaticon.com/authors/vectors-market" title="Vectors Market"> Vectors Market </a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com'</a></div>
+> In this first capstone project, I created a page for a made up conference set to occur in October, 2022. With this project I showcase a cloned version of an original Behance design by Cindy Shin, taking inspiration from her use of colors, patterns and layouts
 
-<div> Icons made by <a href="https://www.freepik.com" title="Freepik"> Freepik </a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com'</a></div>
+## Built With
 
-<div> Icons made by <a href="https://www.flaticon.com/authors/becris" title="Becris"> Becris </a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com'</a></div>
+- HTML
+- CSS
+- Javascript
 
-<div> Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik"> Freepik </a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com'</a></div>
+## Live Demo
+
+
+![alt-text-1](./images/readMe-images/desktop-view.png "title-1") 
+![alt-text-2](./images/readMe-images/mobile-view.png "title-2")     ![alt-text-2](./images/readMe-images/mobile-gif.gif "title-2")
+![alt-text-1](./images/readMe-images/desktop-gif.gif "title-1")
+
```

Comment:
- [ ] Kindly record a five-minute video using loom or any other tool presenting the features of the project you built and share the link to the video. This is for documentation purposes

Reply:
Okay. Thank you. I will do that immediately.